patches/@types+reach__router+1.3.6.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/@types/reach__router/index.d.ts b/node_modules/@types/reach__router/index.d.ts
+index 80faa6e..97213dc 100644
+--- a/node_modules/@types/reach__router/index.d.ts
++++ b/node_modules/@types/reach__router/index.d.ts
+@@ -10,7 +10,9 @@
+ // TypeScript Version: 2.8
+ 
+ import * as React from 'react';
++// @ts-ignore
+ import { Location as HLocation, LocationState } from 'history';
```

Comment:
For some reason build could not find LocationState from history. I checked from history, and it seems to be there. So as a workaround, I applied this patch-package to add the @ts-ignore, which fixes build

Reply:
Was able to delete this patch by removing storybook/addon-notes from core, and replacing it with addon-docs.